### ISSUE TITLE
Performance on standard model

### ISSUE THREAD

**Issue by haofanwang** [wwoods/adversarial-explanations-cifar#1]
Hi, this work seems interesting to me. 

But if I use standard model without adversarial training, it never generates satisfied explanations even I have increased eps or steps. Do you have any suggestions?

**Comment by haofanwang** [comment]
Fixed.

**Comment by wwoods** [comment]
Hi Haofan, sorry for the late reply.  Thanks for the interest though!  Out
of curiosity, what issue did you have?  I'd like to make it as clear as
possible.

**Comment by haofanwang** [comment]
Hi, @wwoods ,

As the algorithm 1 is simple actually, why robust model after adversarial training (without L2-min or robust additions) can generate so semantic adversarial image (like GAN)? Rather than editing some regions, it looks more like a re-construction image.  Even though I removed L2 minimization, it can still work well only with gradient descent. Could you give out more illustrations here?

Besides, I failed to generate semantic adversarial examples on ImageNet with adversarially trained model provided in [Tensorflow](https://github.com/dongyp13/Non-Targeted-Adversarial-Attacks). The generated noise is somewhat semantic, but still much worse than results in your paper (I have tried different params). I haven't train a new robust model, but why other adversarially trained model cannot generate good semantic adversarial images?

**Comment by wwoods** [comment]
Basically, on CIFAR-10, the dimensionality is low enough that it's feasible for adversarial training to provide significant smoothing.  On ImageNet, the domain is much larger (224x224x3), so adversarial training is not as effective as the proposed stochastic Lipschitz constraint.  You will definitely need to train a model with the robust additions to get results comparable to those in the paper.  Ideally I'd pre-train such a model using the code here, but I haven't yet mostly because of time/resource constraints.  Feel free to leave this issue open and hopefully I'll get around to it.

**Comment by haofanwang** [comment]
Thanks, I'll try it later. It's a nice job.
